site/users/index.html.js

```diff
@@ -48,6 +48,16 @@ var logos = [
     img: 'reindex.png',
     link: 'https://www.reindex.io/'
   },
+  {
+    name: 'Meteor',
+    img: 'meteor.png',
+    link: 'https://medium.com/apollo-stack/snappier-uis-with-apollo-client-graphql-bffab0b52b79#.sls7hdvud'
```

Comment:
No problem! I thought perhaps the case study of how we use GraphQL might have been more interesting, but I'll link to Meteor.com instead and remove Apollo.